Using sky-fluid-grid

Blackbaud-BobbyEarl opened the pull request.

Blackbaud-PaulCrowder commented on src/app/about/index.html:
```diff
@@ -1 +1,13 @@
-<my-about></my-about>
+<app-nav role="navigation">
```
For non-Stache sites, I think we should aim to put as much markup in the components themselves rather than in the `index.html` file because then it can be unit tested by simply instantiating the component.  Mind making the change to move the fluid grid markup to the respective components so that the contents of the `index.html` files remain simply `<my-component></my-component>`?

Blackbaud-BobbyEarl commented:
Incorporated your feedback @Blackbaud-PaulCrowder.  Good suggestion.  I also went ahead and updated the versions (still overwritten during `skyux new`) as well as went ahead and incorporated #51.  Also update the learn more link to point to our site instead of GH.

Blackbaud-BobbyEarl commented on package.json:
```diff
@@ -19,9 +19,9 @@
   "license": "ISC",
   "dependencies": {
-    "@blackbaud/skyux": "2.1.0"
+    "@blackbaud/skyux": "2.22.0"
```
These are dynamically updated at `skyux new` time, but I just like to update them when I do a PR.
